window/src/os/windows/window.rs

```diff
@@ -417,6 +431,7 @@ impl Window {
             dead_pending: None,
             saved_placement: None,
             track_mouse_leave: false,
+            last_resize_type: SIZE_RESTORED,
```

Comment:
Not strictly related to this PR, but https://github.com/wez/wezterm/issues/1502 is probably caused by the `nCmdShow` parameter in `WinMain` not being respected when launched via a shortcut with the maximized state.

I don't know (haven't looked) to see if there is a smarter way to know when we want to do that or apply it.

Anyway: doesn't need changes here in this PR, but wanted to float that by you in case you have insights on what to do about it :)

Reply:
I tested and only got problems if I had a configuration setting initial_cols/initial_rows. Anyway, I've made a change in `check_and_call_resize_if_needed` that solves it. I think it's supposed to be that way.